l2lsp/methods/lsp_suggestEnv.go

```diff
@@ -9,7 +9,6 @@ import (
 	l2envpackege "github.com/HexmosTech/lama2/l2env"
 	"github.com/HexmosTech/lama2/l2lsp/request"
 	"github.com/HexmosTech/lama2/l2lsp/response"
-	"github.com/rs/zerolog/log"
 )
 
```

Comment:
Have we tested the VSCode extension with new CLI build? Worth doing it once I think.